sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java

```diff
@@ -40,16 +40,12 @@ public SentryAndroidOptions() {
   }
 
   private @NotNull SdkVersion createSdkVersion() {
-    final SdkVersion sdkVersion = new SdkVersion();
+    final SdkVersion sdkVersion = getSdkVersion() != null ? getSdkVersion() : new SdkVersion();
```

Comment:
Everytime I see a getter called twice I think it might be allocating on each call or otherwise have side effect. Might not be common in Java but it's hard not to wonder (or go look into it).

If it had a `getSdkVersion() ?? new SdkVersion` that would solve, but .. Java.

Is it too much hassle to assing the result of `getSdkVersion()` to test for null and assign?

Reply:
Fixed.